crates/goose/src/agents/skills_extension.rs

```diff
@@ -82,14 +82,18 @@ impl SkillsClient {
         let mut dirs = Vec::new();
 
         if let Some(home) = dirs::home_dir() {
+            // Global skills directories (lowest priority first)
```

Comment:
I don't get the comment here and the one below about priority. Perhaps remove these?